### PR TITLE
Added color by status code #29

### PR DESCRIPTION
**Pull request by ChandanChainani**
image

### CODE REVIEW

**Comment by phra** [comment]
can you attach an example of the output result? thanks!

**Comment by ChandanChainani** [comment]
@phra  I have attached the screenshot

Only added color in dir status code, didn't added the response size.

**Review by phra — CHANGES_REQUESTED** [review]
can you add `console` to to `Cargo.toml`?

**Comment by ChandanChainani** [comment]
@phra Sorry i forgot to include the dependency added as mentioned.

File: src/dirbuster/mod.rs [hunk 1]
```diff
@@ -36,6 +37,25 @@ pub struct DirBuster {
...
 
+fn color_by_status_code(code: &str) -> String {
+    if code.starts_with("1") {
+        return style(code).blue().to_string();
+    }
+    else if code.starts_with("2") {
+        return style(code).green().to_string();
+    }
+    else if code.starts_with("3") {
+        return style(code).magenta().to_string();
```
Comment (phra) [inline src/dirbuster/mod.rs]: I think yellow makes more sense for 3xx

File: src/dirbuster/mod.rs [hunk 2]
```diff
@@ -36,6 +37,25 @@ pub struct DirBuster {
...
+    }
+    else if code.starts_with("4") {
+        return style(code).yellow().to_string();
```
Comment (phra) [inline src/dirbuster/mod.rs]: Maybe orange better fits 4xx

**Review by phra — CHANGES_REQUESTED** [review]

**Comment by ChandanChainani** [comment]
@phra updated 3xx and 4xx status color as mentioned

image

**Review by phra — APPROVED** [review]
LGTM

**Pull request merged**